src/lib/utils/os_utils.cpp

```diff
@@ -163,6 +167,27 @@ uint64_t OS::get_cpu_cycle_counter()
    return rtc;
    }
 
+long OS::get_cpu_total()
+   {
+   static long tt = 0;
+   if(tt == 0)
+#if defined(BOTAN_TARGET_OS_HAS_SYSCONF)
+      tt = sysconf(_SC_NPROCESSORS_CONF);
```

Comment:
Use `::` to refer to functions in the global namespace, ie `::sysconf`

Also, check return value, if <= 0 then fall back to using `std::thread::hardware_concurrency` if available (`HAS_THREADS`) or just return 1